### ISSUE TITLE
Mass importing metadata of PS2 games from IGDB fails

### ISSUE THREAD

**Issue by DennoCoil** [JosefNemec/Playnite#460]
Have a bug.  I have a pack of 50 PS2 games and tried to mass import metadata from IGDB for them.

While it seems like it's downloading metadata, as it goes through the progress bar.  After it finishes, nothing is added or changed.  Individually, I can download the metadata just fine from IGDB.  It's the mass import feature that doesn't work properly.  It works for most other systems and emulated games though, and I haven't seen this happen with other Platforms or tried to put in several other dozen platforms either.

No idea what other conditions it would take to replicate.  My iso files are in zipped GZ files for space saving and for the PS2 emulator to read, the Platform is set to Playstation 2, and I roughly have 10k+ games and counting over several hard drives so far.

**Comment by JosefNemec** [comment]
Could you please go to "About" menu and crate diagnostics package for me? Thank you

**Comment by DennoCoil** [comment]
Went through, added the ISO's again, did the mass meta data, and then a single metadata.
[PS2Fail.zip](https://github.com/JosefNemec/Playnite/files/1688692/PS2Fail.zip)

**Comment by JosefNemec** [comment]
Could share some ISO names for your PS2 games? If there's some special naming scheme then Playnite might not be able to match the records automatically from IGDB database.

Also there's a lot of "OutOfMemory" exceptions in your logs, which doesn't make sense since you have 32GB of RAM. Are you experiencing some other issues with stability?

**Comment by DennoCoil** [comment]
[Uploading PS2Fail.zip…]()

**Comment by DennoCoil** [comment]
I think we found the problem.  The names of the files also includes stuff in between parentheses ( ) and possibly brackets [ ].  My PS2 games are some of the ones that are named like this and Playnite is intelligent enough to ignore things like (World) and (Unl) or other GoodMerged naming schemes when adding in games.

However, when adding in uncompressed ISO files to Playnite from the PS2 games, it puts in the game's name and then has "iso" right after it.  Like this:  "WhatEverGame iso".  I used uncompressed images now because after some testing, the PS2 emulator can't read GZ files, even though people said it should.  That may be it and Playnite just adds in the "iso" when they're straight uncompressed images.  

I noticed that Playnite\IGDB metadata download regularly fails when colons are used instead of hyphens. Don't know why they're named with colons either, but that's how they came up when populated on the list.  IGDB's search engine may not be intelligent enough to ignore or combine certain characters together.

As for the "OutOfMemory" issue, I couldn't say.  It doesn't use a whole lot of memory or crash because of that.  Could potentially be from using a lot of things at once that tries to hit the limit and I had to force close some things.
